packages/viewer/src/components/scene-tree-table-cell/scene-tree-table-cell.spec.tsx

```diff
@@ -545,6 +552,31 @@ describe('<vertex-scene-tree-table-cell>', () => {
 
     expect(hovered).toHaveBeenCalledWith(undefined);
   });
+
+  it('blurs the element when an action button is clicked', async () => {
+    const node = createNode({ selected: true });
+    const { cell } = await newComponentSpec({
+      html: `
+        <vertex-scene-tree-table-cell expand-toggle isolate-button visibility-toggle></vertex-scene-tree-table-cell>
+      `,
+      node,
+    });
+
+    const expand = cell.shadowRoot?.querySelector('.expand-btn');
+    const visibility = cell.shadowRoot?.querySelector('.visibility-btn');
+    const isolate = cell.shadowRoot?.querySelector('.isolate-btn');
+
+    console.log(expand);
```

Comment:
Nit: do you still want this?

Reply:
I do not, didn't mean to leave that in 🙃 - removed!